app/views/houses/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <h1>The houses</h1>
-<% houses.each do |house| %>
+<% @house.each do |house| %>
   <div class="house">
-    <h2><a href="houses_path"</a>@house</h2>
-    <img src="<% house.image_url %>"/>
-  <div>
-<% end %>
\ No newline at end of file
+    <h5><%= link_to house.name, house  %></h5>
+     <img src="<%= house.img_url %>">
+  </div>
+<% end %>
```

Comment:
👍  Line 2 should remain as `@houses.each do`
But, good catch on line 4 with the path and also on line 6 with the closing `</div>`